core/ir/aarch64/instr_create.h

```diff
@@ -556,6 +556,10 @@
     instr_create_1dst_1src((dc), OP_mrs, (Xt), (sysreg))
 #define INSTR_CREATE_msr(dc, sysreg, Xt) \
     instr_create_1dst_1src((dc), OP_msr, (sysreg), (Xt))
+#define INSTR_CREATE_sys(dc, op1, CRn, CRm, op2, Rn) \
+    instr_create_0dst_5src(dc, OP_sys, op1, CRn, CRm, op2, Rn)
```

Comment:
We want `DC` instructions to have a memory operand, not a plain register.  That could be done either by having encoding variants of `OP_sys`: some take a register, some a base-disp (and the encoder extracts the base, just like for `OP_ldr`).  Or, we could split the opcodes, having `OP_dc` or `OP_sys_dc` or sthg.  Either way, we probably want `INSTR_CREATE_dc` which takes in a base-disp operand.